systemtest/src/test/java/io/strimzi/systemtest/specific/RackAwarenessIsolatedST.java

```diff
@@ -0,0 +1,421 @@
+
+        KafkaConnectResource.replaceKafkaConnectResourceInSpecificNamespace(clusterName, kafkaConnect -> {
+            kafkaConnect.getSpec().setRack(new Rack(topologyKey));
+        }, clusterOperator.getDeploymentNamespace());
+        KafkaConnectUtils.waitForConnectReady(clusterOperator.getDeploymentNamespace(), clusterName);
+        LOGGER.info("KafkaConnect is ready with changed rack key: '{}'.", topologyKey);
+        LOGGER.info("Verify KafkaConnect rack key update");
+        kc = KafkaConnectResource.kafkaConnectClient().inNamespace(clusterOperator.getDeploymentNamespace()).withName(clusterName).get();
+        assertThat(kc.getSpec().getRack().getTopologyKey(), is(topologyKey));
+
+        final String scraperPodName = ScraperUtils.getScraperPod(clusterOperator.getDeploymentNamespace()).getMetadata().getName();
+
+        List<String> kcPods = kubeClient().listPodNames(clusterOperator.getDeploymentNamespace(), clusterName, Labels.STRIMZI_KIND_LABEL, KafkaConnect.RESOURCE_KIND);
+        KafkaConnectUtils.sendReceiveMessagesThroughConnect(kcPods.get(0), TOPIC_NAME, kafkaClientsPodName, scraperPodName, clusterOperator.getDeploymentNamespace(), clusterName);
+
+        // check the ClusterRoleBinding annotations and labels in Kafka cluster
+        Map<String, String> actualLabel = KafkaConnectResource.kafkaConnectClient().inNamespace(clusterOperator.getDeploymentNamespace()).withName(clusterName).get().getSpec().getTemplate().getClusterRoleBinding().getMetadata().getLabels();
+        Map<String, String> actualAnno = KafkaConnectResource.kafkaConnectClient().inNamespace(clusterOperator.getDeploymentNamespace()).withName(clusterName).get().getSpec().getTemplate().getClusterRoleBinding().getMetadata().getAnnotations();
+
+        assertThat(actualLabel, is(label));
+        assertThat(actualAnno, is(anno));
+
+        // revert changes for CO deployment
+        clusterOperator.unInstall();
+        clusterOperator = new SetupClusterOperator.SetupClusterOperatorBuilder()
+                .withExtensionContext(BeforeAllOnce.getSharedExtensionContext())
+                .withNamespace(clusterOperator.getDeploymentNamespace())
+                .createInstallation()
+                .runBundleInstallation();
+
+        DeploymentUtils.waitTillDepHasRolled(clusterOperator.getDeploymentNamespace(), ResourceManager.getCoDeploymentName(), 1, coSnapshot);
+    }
+
+    @IsolatedTest
+    void testMirrorMaker2RackAwareness(ExtensionContext extensionContext) {
+        assumeFalse(Environment.isNamespaceRbacScope());
+
+        TestStorage storage = new TestStorage(extensionContext);
+        String namespace = storage.getNamespaceName();
+        String clusterName = storage.getClusterName();
+        String kafkaClusterSourceName = clusterName + "-source";
+        String kafkaClusterTargetName = clusterName + "-target";
+        String topologyKey = "kubernetes.io/hostname";
```

Comment:
Can you create some constant or something for this? It's repeated in every test.

Reply:
Yes, that's a good idea.